examples/graphql/src/__tests__/acceptance/graphql-context.acceptance.ts

```diff
@@ -34,7 +34,7 @@ describe('GraphQL context', () => {
     server.resolver(RecipeResolver);
 
     // Customize the GraphQL context with additional information for test verification
-    server.bind(GraphQLBindings.GRAPHQL_CONTEXT_RESOLVER).to(ctx => {
+    server.bind(GraphQLBindings.GRAPHQL_CONTEXT_RESOLVER).to(async ctx => {
       return {...ctx, meta: 'loopback'};
     });
```

Comment:
It's because the `ContextFunction` returns a `Promise` in Apollo 4: https://github.com/apollographql/apollo-server/blob/main/packages/server/src/externalTypes/context.ts#L8-L11